java/ingest/ingest-runner/src/main/java/sleeper/ingest/impl/IngestCoordinator.java

```diff
@@ -285,11 +289,11 @@ public CompletableFuture<IngestResult> asyncCloseReturningResult()
                             .flatMap(List::stream).collect(Collectors.toList());
                     IngestResult result = IngestResult.fromReadAndWritten(recordsRead, filesWritten);
                     long noOfRecordsWritten = result.getRecordsWritten();
-                    double elapsedSeconds = (System.currentTimeMillis() - ingestCoordinatorCreationTime) / 1000.0;
-                    METRICS_LOGGER.info(String.format("Wrote %d records to S3 in %.1f seconds at %.1f per second",
+                    LoggedDuration duration = LoggedDuration.between(ingestCoordinatorCreationTime, Instant.now());
+                    METRICS_LOGGER.info(String.format("Wrote %d records to S3 in %s seconds at %.1f per second",
```

Comment:
This String.format call seems inconsistent with the rest of the logs. It also looks like it will output "seconds" twice.